### ISSUE TITLE
`npx dalai llama install 7B` installs 34 GB data in home directory under `~/dalai`

### ISSUE THREAD

**Issue by paulmenzel** [cocktailpeanut/dalai#440]
Running `npx dalai llama install 7B` it creates a directory `"$HOME"/dalai` with 34 GB of content.

Is it possible to configure a different directory?

**Comment by noorus** [comment]
Yes, as the homepage states, you can pass `--home <path>` to the `install` command to change it.
